prelude/__internal__/Random.spec.mad

```diff
@@ -0,0 +1,190 @@
+import Float from "Float"
+import Math from "Math"
+import { Just } from "Maybe"
+import String from "String"
+
+import {
+  LCG_A,
+  LCG_C,
+  LCG_M,
+  SEED_MAX,
+  SEED_MIN,
+  Seed,
+  apply,
+  generate,
+  generateFromString,
+  intGradient,
+  iterateBinary,
+  mkSeed,
+  mkSeedFromString,
+  modClamp,
+  next,
+  perturbation,
+  seededShuffle,
+  stepN,
+  unSeed,
+} from "./Random"
+import { assertEquals, test } from "./Test"
```

Comment:
Same thing here

Reply:
Oh yeah, as above I think I misunderstood what you meant. We should probably update that one to "Test" yeah.